### PR TITLE
Translate FAQs into Hindi, Bahasa and Romanian

### PR DESCRIPTION
**Pull request by arzoo14**
Currently, there are three FAQs which are not in Hindi, Bahasa
and Romanian language. This translates those FAQs.

Fixes https://github.com/coala/projects/issues/768

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: data/locale/hi/faq.json [hunk 1]
```diff
@@ -11,5 +11,8 @@
   "mentor-obligations.md": "GSoC के दौरान एक उपदेशक के रूप में मेरे दायित्व क्या हैं?",
   "thesis.md": "मैं coala के साथ थीसिस लिखना चाहता हूं!",
   "application-template.md": "क्या आपके पास कोई एप्लिकेशन टेम्प्लेट है?",
-  "project-participation.md": "क्या सभी सूचीबद्ध परियोजनाएं GSoC (या कोई अन्य पहल) का हिस्सा होगी?"
-}
+  "project-participation.md": "क्या सभी सूचीबद्ध परियोजनाएं GSoC (या कोई अन्य पहल) का हिस्सा होगी?",
+  "cep-and-its-importance.md": "CEP क्या है? पूर्व-GSoC विश्लेषण करना और प्रस्ताव को जल्द से जल्द डिजाइन करना इतना महत्वपूर्ण क्यों है?",
+  "coala-will-apply-for-gsoc.md": "क्या coala अगले साल गूगल समर ऑफ कोड के लिए आवेदन करेगा?",
+  "high-involvement-status.md": "किसी भी परियोजना की उच्च भागीदारी की स्थिति का क्या अर्थ है?"
+}
```
Comment (KVGarg) [inline data/locale/hi/faq.json]: End a file with a newline. Please add it, in this file and in other changed files too.

Comment (arzoo14) [reply data/locale/hi/faq.json]: Sure. I'll do the same by evening.

**Comment by arzoo14** [comment]
```
raise IOError(constants.ERROR_DATA_FILE_ABSENT % the_file)
OSError: File coala.yaml does not exist
The command ".ci/check_moban.sh" exited with 1.
```
@KVGarg 
This error is coming in Moban. I have tried many things but none of them worked. Plz help.

**Comment by KVGarg** [comment]
> This error is coming in Moban. I have tried many things but none of them worked. Plz help.

This shouldn't happen but no worries I will check it once I've some free time. Else you can ask about this error on coala main gitter channel. 
Also, the commits needs to be squashed to 1 commit.

**Comment by arzoo14** [comment]
> > This error is coming in Moban. I have tried many things but none of them worked. Plz help.
> 
> This shouldn't happen but no worries I will check it once I've some free time. Else you can ask about this error on coala main gitter channel.
> Also, the commits needs to be squashed to 1 commit.

I have squashed the commits into one.

**Comment by abhishalya** [comment]
Can you rebase this PR?

**Pull request closed without merging**